server/models/Message.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require("mongoose");
+
+const messageSchema = new mongoose.Schema(
```

Comment:
It would be nice to have a read status for messages :)

Eg: 
```
read: {
type: Boolean
default: false
}
```

Reply:
Oh, that I good idea, Added, Thanks